policyengine_canada/variables/household/income/individual/other_than_eligible_taxable_dividend_income.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_canada.model_api import *
+
+
+class other_than_eligible_taxable_dividend_income(Variable):
```

Comment:
What do you mean by eligible? Taxable and tax-exempt are the typical categories.
```suggestion
class tax_exempt_dividend_income(Variable):
```